From 2190ddeef7b0f43ef38512915ba4a666e844d39c Mon Sep 17 00:00:00 2001
Message-Id: <2190ddeef7b0f43ef38512915ba4a666e844d39c@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Fri, 20 Jul 2018 07:52:12 -0400
Subject: [PATCH] tools: Fix typo generating adapter_wwpn field

https://bugzilla.redhat.com/show_bug.cgi?id=1601377

Fix typo from commit id d45bee449 for the parent_wwpn field
resulting in parent_wwnn being printed twice.

Signed-off-by: John Ferlan <jferlan@redhat.com>
Reviewed-by: Erik Skultety <eskultet@redhat.com>
(cherry picked from commit 313eaae3b5a4f9b1e94b153759b878614caecf34)
Reviewed-by: Erik Skultety <eskultet@redhat.com>
---
 tools/virsh-pool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/virsh-pool.c b/tools/virsh-pool.c
index cc49a5b96d..6faff781b2 100644
--- a/tools/virsh-pool.c
+++ b/tools/virsh-pool.c
@@ -371,7 +371,7 @@ virshBuildPoolXML(vshControl *ctl,
             if (adapterParent)
                 virBufferAsprintf(&buf, " parent='%s'", adapterParent);
             else if (adapterParentWwnn && adapterParentWwpn)
-                virBufferAsprintf(&buf, " parent_wwnn='%s' parent_wwnn='%s'",
+                virBufferAsprintf(&buf, " parent_wwnn='%s' parent_wwpn='%s'",
                                   adapterParentWwnn, adapterParentWwpn);
             else if (adapterParentFabricWwn)
                 virBufferAsprintf(&buf, " parent_fabric_wwn='%s'",
-- 
2.18.0

